static/js/components/tiles/ranking_tile.tsx

```diff
@@ -99,22 +103,32 @@ export function RankingTile(props: RankingTilePropType): JSX.Element {
     chartWidth: number,
     chartHeight: number,
     chartHtml: string,
-    rankingPoints: RankingPoint[],
-    sources: string[],
-    svNames: string[]
+    chartTitle: string,
+    sources: string[]
   ): void {
     embedModalElement.current.show(
       "",
-      rankingPointsToCsv(rankingPoints, svNames),
+      () => {
+        const perCapitaVariables = props.variables
+          .filter((v) => v.denom)
+          .map((v) => v.statVar);
+        return datacommonsClient.getCsv({
```

Comment:
ranking tile should also have a date? & this tile also uses the same special date logic as map and scatter

Reply:
Good catch- added